modules/exporter-common/src/main/scala/io/janstenpickle/trace4cats/export/ExportRetryConfig.scala

```diff
@@ -1,9 +1,18 @@
 package io.janstenpickle.trace4cats.`export`
 
+import cats.kernel.Eq
+import cats.derived.semiauto
+
 import scala.concurrent.duration._
 
 case class ExportRetryConfig(
   delay: FiniteDuration = 500.millis,
   nextDelay: FiniteDuration => FiniteDuration = _ + 100.millis,
   maxAttempts: Int = 100
 )
+
+object ExportRetryConfig {
+  implicit val nexDelayEq: Eq[FiniteDuration => FiniteDuration] = Eq.fromUniversalEquals
```

Comment:
Wait. You can't compare functions.

Reply:
and there's a typo in the name `nexDelayEq`

Reply:
I think this should work if one makes a new instance of config as `oldConfig.copy(delay = newDelay)`

Reply:
But in general, we can't compare functions. So we may create a new resource even if it's not needed. Which is probably not so bad (compared to the opposite)

Reply:
Ah, I've got an idea for this